reset.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Q: What's this file for.
No mention of it in the PR description

Reply:
Mistake.  Closing this PR.